changelog_v0.1.md

```diff
@@ -566,7 +566,7 @@ This will be a convention throughout the language starting from here. Connected
 
 In v0.09 the module pattern had a rich panoply of available patterns, most of which were used once or twice.
```

Comment:
I think I am exaggerating that `most` patterns were used once or twice. This should be `some`.

Reply:
Ah, I was reffering to module patterns in particular. Maybe I should replace this with....

```
In v0.09 the module grammar had a rich panoply of available patterns, most of which were used once or twice.
```